cpp/ql/lib/semmle/code/cpp/models/interfaces/Sql.qll

```diff
@@ -0,0 +1,30 @@
+/**
+ * Provides abstract classes for modeling functions that execute and escape SQL query strings.
+ * To use this QL library, create a QL class extending `SqlExecutionFunction` or `SqlEscapeFunction`
+ * with a characteristic predicate that selects the function or set of functions you are modeling.
+ * Within that class, override the predicates provided by the class to match the way a
+ * parameter flows into the function and, in the case of `SqlEscapeFunction`, out of the function.
```

Comment:
This describes how to extend the library, not really how to "use" it, in my opinion.

Reply:
True. But the purpose of this interface is mainly that other users can extend it, and our SQL injection query will magically make use of those new implementations.

Reply:
I do agree, though. If the purpose is to create an interface that other people can write new queries against, this QLDoc (and many of the other QLDocs in the `models/interfaces` folder) aren't sufficiently.

Reply:
I'd still prefer a rephrase from "To use this QL library" -> "To extend this QL library" perhaps?

Reply:
Good idea. Fixed in bd5edc7ae5c5f36257cc4bd0a413c7bd50d6b752.